### DISCUSSION TITLE
Hover effect missing on buttons

### DISCUSSION THREAD

**Discussion by gagan-bhullar-tech** [withastro/starlight#2090] [Feature Requests]
### What version of `starlight` are you using?

0.24.0

### What version of `astro` are you using?

4.5.12

### What package manager are you using?

pnpm

### What operating system are you using?

Mac

### What browser are you using?

Chrome

### Describe the Bug

Actions buttons does not have hover effect.

- text: Get Started
      link: /getting-started
      icon: right-arrow
      variant: primary

### Link to Minimal Reproducible Example

""

### Participation

- [X] I am willing to submit a pull request for this issue.

**Comment by delucis** [comment]
Hi @gagan-bhullar-tech! Thanks for the feedback. As this isn’t a bug but effectively a feature request, I’ll move it to our discussions area.

**Comment by HiDeoo** [comment]
Thanks for the feedback.

This seems to be a duplicate of https://github.com/withastro/starlight/discussions/1912 so I'm going to close this discussion so we can keep the conversation in one place :raised_hands:
